skimage/filters/_gabor.py

```diff
@@ -94,6 +95,74 @@ def gabor_kernel(frequency, theta=0, bandwidth=1, sigma_x=None, sigma_y=None,
     return g
 
 
+def morlet_kernel(frequency, theta=0, bandwidth=1,
+                      sigma_x=None, sigma_y=None, n_stds=3, offset=0):
+    """Return complex 2D morlet filter kernel.
+
+       Morlet kernel is a Gabor kernel with no DC offset.
```

Comment:
I would add "This means that not only the imaginary part (`sin` component), but also the real part (`cos` component) have zero sum."